doc/etc/config.ini

```diff
@@ -69,8 +69,8 @@ write_index_ms = 60000
 #                       enables outputting DMQ channel throughput statistics to
 #                       the console.
 
-[Stats]
-console_stats_enabled   = true
+; [Stats]
+; console_stats_enabled   = true
```

Comment:
I think you can just remove this entire section.

Reply:
I guess it doesn't hurt to have it explicit since is a file that's supposed to be documentation too.

Reply:
I'll do although tot entirely related here ;).